### ISSUE TITLE
ckb K70 LUX Sierra problem

### ISSUE THREAD

**Issue by DeviousR** [ccMSC/ckb#498]
My ckb won't work. It says that Xcode isn't running properly and says I have to do something with the license agreement. PLEASE HELP

**Comment by mhennessie** [comment]
CKB causes lots of issues in Sierra. I suggest not using it.

**Comment by atarikguney** [comment]
@DeviousR Don't waste your time. There is a solution to that problem, but it does not help eventually. If you want to change something with your keyboard, make the change via a Windows computer, and persist your changes inside the keyboard by saving them into the keyboard itself, which is the default option for most of the possible operations. Then, continue on Mac.

**Comment by Ferannor** [comment]
Agree, I compiled it on Sierra, add a fix around on xcode 8 but compatibility is really tricky. 
Apple has change some API and there to many problems to use it for now. So forget it at this time.
